.github/workflows/docs.yml

```
@@ -75,8 +77,8 @@ jobs:
         id: doc_coverage
         run: |
           echo "${{ github.event.pull_request.base.sha }}"
-          docstr-coverage --skip-magic --skip-init --fail-under=0 --docstr-ignore-file=base/.docstr_coverage base/pyccel base/ci_tools 2>&1 | tee base_cov
-          docstr-coverage --skip-magic --skip-init --fail-under=0 --docstr-ignore-file=compare/.docstr_coverage compare/pyccel compare/ci_tools 2>&1 | tee compare_cov
+          docstr-coverage --skip-magic --skip-init --fail-under=0 --exclude=base/pyccel/extensions --docstr-ignore-file=base/.docstr_coverage base/pyccel base/ci_tools 2>&1 | tee base_cov
+          docstr-coverage --skip-magic --skip-init --fail-under=0 --exclude=compare/pyccel/extensions --docstr-ignore-file=compare/.docstr_coverage compare/pyccel compare/ci_tools 2>&1 | tee compare_cov
```

Maybe try:
```suggestion
          docstr-coverage --skip-magic --skip-init --fail-under=0 --exclude=".*base/pyccel/extensions" --docstr-ignore-file=base/.docstr_coverage base/pyccel base/ci_tools 2>&1 | tee base_cov
          docstr-coverage --skip-magic --skip-init --fail-under=0 --exclude=".*compare/pyccel/extensions" --docstr-ignore-file=compare/.docstr_coverage compare/pyccel compare/ci_tools 2>&1 | tee compare_cov
```
so the regex matches the absolute path?